### ISSUE TITLE
Importing Vice or 64tass labels always fails

### ISSUE THREAD

**Issue by gsanjin** [slajerek/RetroDebugger#28]
I am not able to import labels in latest available version, neither vice labels nor 64tass labels. I am not using Java and KickAssembler. If I try to import vice labels I get message:

"Loading lables failed: Error: Found whitespace in your keyname (use quotes to include) at line 1m2 >> al 2860 .dec2b92 al"

Vice labels are in format:
```
al 2860 .dec2B92
al 38b5 .a3374
al 1865 .chkScrMovingUD
al 1873 .chkScrMovingUD:_l1
al 1886 .chkScrMovingUD:_l2
al 1899 .chkScrMovingUD:_l3
al 18ac .chkScrMovingUD:_l4
al 18ba .chkScrMovingUD:_exit
al 149e .touchingLt
al dbb .MainGameLoop
...
```

so, just as specified in old documentation where white space also exists. If I prepare same labels in 64tass format, I get another error message:

"Loading labels failed. Error: Found EOF while looking for a key name (check your syntax).

and labels are in format
```
dec2B92		= $2860
a3374		= $38b5
tblBoxAnimState	= $fb36
chkScrMovingUD	= $1865
touchingLt	= $149e
MainGameLoop	= $0dbb
...
``` 
so, in same format as old RetroDebugger docs are specifying. Tried converting CR/LF to LF but getting same errors. Labels are generated by 64tass and generated vice labels work normally in latest Vice, but that's not debugger ... Since automatic loading of labels is not supported for CRT files, I am always trying to manually import them.

BTW, please put clean link to your Paypal donation page, as current link is being blocked by security/adblocking software, and it's impossible to open that link because of viglink. Understand this is being blocked on very sensitive point - payments ;) Cheers!

**Comment by slajerek** [comment]
That is strange behaviour, could you please post or send me privately these label files? 

ps: as for PayPal, the direct link is: https://www.paypal.com/donate?token=FsyY3dQkm4PdY0yT0irRzby3rBzbKUoOF8g61BcmWVR23uka6JHdlDusppesSWoriWPrK8vNr6H6_Pws&locale.x=GB

**Comment by gsanjin** [comment]
Thank you for quick reply - I think I managed to send you privately those label files.

**Comment by gsanjin** [comment]
My mistake: I used custom extension for 64tass labels (*.labels or *.txt). When I changed extension to supported formats all is loading ok, through menu or command line arg.

![image](https://github.com/slajerek/RetroDebugger/assets/28846715/ca24730d-45f7-4866-bb43-bdf956fc01e2)

Looks like 64tass assembler doesn't generate RetroDebugger compatible vice-labels format, but 64tass format works properly, just give a labels file supported file extension :)

Thanks Slajerek for help with this!
